### DISCUSSION TITLE
Mystic Biomes

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#912] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/mystics-biomes
+ 7 biomes
+ 4 trees
+ 1 cactus

**Comment by roaxyane** [comment]
yes please! bump!

**Comment by atinyfroge** [comment]
I'd give you all my dino nuggets for this to happen 🙏
